Analytics/importers/scheduler_new.py

```diff
@@ -10,15 +10,17 @@
 from apscheduler.jobstores.sqlalchemy import SQLAlchemyJobStore
 from retrying import retry
 
-import settings
+
 from importers.state_decorator import ImporterStatus
 from models.importer_status import ImporterStatuses
 from models.api import API as Api_Class
 from app import create_app
 from db import db
+sys.path.append('..')
+import settings
```

Comment:
Is there something functionally stopping settings from being above sys.path.append? if not please swap. If so, please separate them from the other imports

Reply:
sys.path.append has to come before import settings. I've separated the imports